src/io/read_tree_lhalo_hdf5.c

```diff
@@ -260,32 +268,36 @@ int setup_forests_io_lht_hdf5(struct forest_info *forests_info, const int firstf
        unique galaxy indices (across all files, all trees and all tasks) for this run*/
     run_params->FileNr_Mulfac = 1000000000000000LL;
     run_params->ForestNr_Mulfac = 1000000000LL;
-    
+
+    /* Finally setup the multiplication factors necessary to generate
+       unique galaxy indices (across all files, all trees and all tasks) for this run*/
+    run_params->FileNr_Mulfac = 1000000000000000LL;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Variable 'run_params->FileNr_Mulfac' is reassigned a value before the old one has been used.](https://app.codacy.com/manual/jacobseiler/sage-model/pullRequest?prid=4620074)